src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -453,11 +453,11 @@ export const TradeInput = ({ isCompact, tradeInputRef, onChangeTab }: TradeInput
       <SharedTradeInput
         bodyContent={bodyContent}
         footerContent={footerContent}
-        shouldOpenSideComponent={hasUserEnteredAmount}
         headerRightContent={headerRightContent}
         isCompact={isCompact}
         isLoading={isLoading}
-        sideComponent={CollapsibleQuoteList}
+        SideComponent={CollapsibleQuoteList}
+        shouldOpenSideComponent={hasUserEnteredAmount}
```

Comment:
👀

Reply:
yeah needs title case to avoid `Property 'sideComponent' does not exist on type 'JSX.IntrinsicElements'` error in react :D